esp-wifi/src/ble/os_adapter_esp32s3.rs

```diff
@@ -201,11 +204,18 @@ pub(crate) fn create_ble_config() -> esp_bt_controller_config_t {
 
 pub(crate) unsafe extern "C" fn interrupt_on(intr_num: i32) {
     trace!("interrupt_on {}", intr_num);
-    unsafe { crate::hal::xtensa_lx::interrupt::enable_mask(1 << 1) };
+    unwrap!(interrupt::enable(
```

Comment:
@bjoernQ I thought the blobs only dealt in CPU interrupts (0-32), not peripheral interrupt numbers - could you check if this is okay?

Reply:
Those are indeed peripheral interrupts - IIRC the original code was from the time when we manually mapped peripheral interrupts to CPU interrupts (before the vectoring) - so this is good

Reply:
Lovely, thanks!